.zuul.yaml

```diff
@@ -18,90 +25,42 @@
-                          disabled: true
-                        prometheusConfig:
-                          replicas: 1
-                          enableRemoteWriteReceiver: true
-                          scrapeInterval: 30s
-                          persistentVolumeClaim:
-                            resources:
-                              requests:
-                                storage: 20G
-                        resourceSelector:
-                          matchLabels:
-                            service: metricStorage
-                        retention: 24h
-            target:
-              kind: OpenStackControlPlane
-          - patch: |-
-              - op: remove
-                path: /spec/telemetry/template/metricStorage/monitoringStack
-            target:
-              kind: OpenStackControlPlane
-      cifmw_install_yamls_whitelisted_vars: &install_yamls_whitelist
-        - 'WATCHER_REPO'
-        - 'WATCHER_BRANCH'
-        - 'OUTPUT_DIR'
-      pre_deploy_create_coo_subscription:
-        - name: Deploy cluster-observability-operator
-          type: playbook
-          source: "{{ watcher_coo_hook }}"
-      cifmw_tempest_tempestconf_config:
-        overrides: |
-          compute.min_microversion 2.56
-          compute.min_compute_nodes 2
-          placement.min_microversion 1.29
-          compute-feature-enabled.live_migration true
-          compute-feature-enabled.block_migration_for_live_migration true
-          service_available.sg_core true
-          telemetry_services.metric_backends prometheus
-          telemetry.disable_ssl_certificate_validation true
-          telemetry.ceilometer_polling_interval 15
-          optimize.datasource prometheus
-          optimize.openstack_type podified
-          optimize.proxy_host_address {{ hostvars['controller']['ansible_host'] }}
-          optimize.proxy_host_user zuul
-          optimize.prometheus_host metric-storage-prometheus.openstack.svc
-          optimize.prometheus_ssl_enabled true
-          optimize.prometheus_ssl_cert_dir /etc/prometheus/secrets/combined-ca-bundle
-          optimize.podified_kubeconfig_path /home/zuul/.crc/machines/crc/kubeconfig
-          optimize.podified_namespace openstack
+      cifmw_extras:
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].
+           src_dir }}/scenarios/centos-9/multinode-ci.yml"
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].
+           src_dir }}/scenarios/centos-9/horizon.yml"
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/watcher-operator'].
+           src_dir }}/ci/scenarios/edpm.yml"
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/watcher-operator'].
+           src_dir }}/ci/tests/watcher-master.yml"
+
+- job:
+    name: periodic-watcher-operator-validation-master
```

Comment:
technically we don't need this job in fr2 since it doesn't run in check. Since it's identical to the main version it should be ok, however